cirq/contrib/two_qubit_gates/gate_compilation.py

```diff
@@ -0,0 +1,379 @@
+"""Attempt to tabulate single qubit gates required to generate a target 2Q gate
+with a product A k A."""
+from functools import reduce
+from typing import Tuple, Sequence, List, NamedTuple
+
+import numpy as np
+import attr
```

Comment:
I don't think we have `attr` listed as a requirement anywhere. Maybe it's an indirect dependency?

I recommend using `@dataclass` instead, as that has been standardized in python 3.7.

Reply:
Done